webapp/store/index.js

```diff
@@ -51,9 +51,16 @@ export default async () => {
     whitelist: getPersistWhiteList()
   };
 
+  // persistReducer will break the store if there are no
+  // plugin reducers to pass it so set to null if getPluginReducers()
+  // doesn't pass anything
+  const pluginReducers = getPluginReducers()
```

Comment:
Why call this function twice? Is there a way to do it without calling it twice?

Reply:
Will clean up. Using `let` and replacing the variable should work.